### PR TITLE
Readme and dependency updates

### PR DESCRIPTION
**Pull request by alexanderkirtzel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Become independent from locked-in analytics systems and set up reliable tracking
 
 ## 🤓 Usage
 
-You can implement all sorts of front-end user events easily with walker.js. From e-commerce actions like product add to carts or order complete events to product and UX events like navigation, or filter usage, etc.
+You can implement all sorts of front-end user events easily with walker.js. From product and UX events like "newsletter signup", or filter usage, etc. to e-commerce actions like product add to carts or order complete events. It's up to you what you want to measure!
```
Comment (aylaju) [inline README.md]: I would cancel the last sentence, should be very clear from the ones before and only adds more text

File: README.md [hunk 2]
```diff
@@ -124,8 +126,8 @@ Or as a script
 By using the walker.js you don't have to deal with event listener or mutation observer initialization anymore. The walker comes with a bunch of integrated triggers that will fire your event at the right moment.
 
 ```html
-<!-- The trigger will fire the "product action" event -->
-<b data-elb="product" data-elbaction="TRIGGER:action">...</b>
+<!-- The trigger will fire the "app login" event -->
```
Comment (aylaju) [inline README.md:129]: Newsletter signup event?

**Review by aylaju — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alexanderkirtzel) [reply README.md:129]: It's another example on purpose here

**Review by aylaju — APPROVED** [review]
Amazing

**Pull request merged**